recipes-bsp/bootfiles/rpi-cmdline.bb

```diff
@@ -9,7 +9,9 @@ inherit deploy nopackages
 CMDLINE_DWC_OTG ?= "dwc_otg.lpm_enable=0"
 
 CMDLINE_ROOT_FSTYPE ?= "rootfstype=ext4"
-CMDLINE_ROOTFS ?= "root=/dev/mmcblk0p2 ${CMDLINE_ROOT_FSTYPE} rootwait"
+CMDLINE_ROOT_PARTITION ?= "/dev/mmcblk0p2"
```

Comment:
We will also need docs for all these with examples. Can you add a docs commit too?

Reply:
Yes! I'll add the document in docs/extra-build-config.md